services/code_hosting/git_hub_service.rb

```diff
@@ -12,6 +14,12 @@ class GitHubService < CodeHostingService
 
     class << self
       attr_accessor :status_context_prefix
+
+      def self.temp_path
```

Comment:
I wonder if it makes sense to have this live as some constant in the fastlane.ci app. I think we need access to the temp directory in a few places, so it might be a good thing to extract.

Reply:
The temp path is live as long as any `GithubService` access to the class variable (as it creates the directory in case it doesn't exist). It comes in my mind the idea of implementing some kind of _global_ (between `GithubService` instances) state where some kind of information might be cached. So maybe we don't want to be cloning the same repo over and over again, and its less expensive to have some kind of metadata along with the cloned repo as long as the path exists (i.e., date of clone, cache of `Fastfile.json`).